serveradmin/powerdns/migrations/0001_powerdns_default_schema.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.8 on 2020-06-03 07:58
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        # Schema for Power DNS version 4.1.x as available on Github:
+        # https://github.com/PowerDNS/pdns/blob/rel/auth-4.1.x/modules/gpgsqlbackend/schema.pgsql.sql
+        migrations.RunSQL(sql="""
```

Comment:
We can't add one here. The statements all run with `CREATE TABLE IF NOT EXISTS` because they should only be created if this is newly setup PowerDNS server. If I would provide a roll back operation and Serveradmin is attached to an existing one it would delete tables of an already running PowerDNS server which would be fatal.